### ISSUE TITLE
China mobile phone format is incorrect, in :national format

### ISSUE THREAD

**Issue by ledowong** [floere/phony#406]
And I don't know how to fix, so just FYI:

https://en.wikipedia.org/wiki/Telephone_numbers_in_China#Calling_formats

Beijing phone (not mobile): +861065132266
national format is 01065132266, which is correct, trunk('0')

However when it is a mobile phone number, which must start with 1, and > 13, for example:
+8613771044821
phony format this to 013771044821, which is wrong, should be 13771044821, no trunk('0')

so basically the trunk('0') is conditional base on the phone number. I traced the code, seems the DSL not able to config that.

**Comment by mrinterweb** [comment]
I've noticed this for 10 digit Chinese phone numbers. There are certain provinces where 10 digit phone numbers are valid. Phony sees these valid phone numbers as invalid. 

I've come up with a hack to bypass phony's validation until this is resolved:

```
validates :phone_number, phony_plausible: true, unless: -> { chinese_phone_number?(phone_number) }

private

    def chinese_phone_number?(phone)
      phone.present? && phone.match(/^\+?86/)
    end
```
